### ISSUE TITLE
Better ensemble.autocorrelation_time

### ISSUE THREAD

**Issue by evanberkowitz** [evanberkowitz/supervillain#94]
In #79 I mentioned that it would be nice to be able to provide a cut before computing the autocorrelation time.  This really ought to wait until after #80.  To have a truly fully-featured `autocorrelation_time` method it would be great to be able to

 - [x] cut from the beginning for thermalization
 - [ ] specify means if known.  For example, `XWrapping` has a known mean of 0 by symmetry; if an ensemble got trapped it might falsely look decorrelated.
